django/researchdata/views.py

```diff
@@ -10,11 +10,134 @@
                cwb_output_ngrams)
 
 
-class MonolingualCorporaInputView(TemplateView):
+class InputMonolingualView(TemplateView):
     """
-    Class-based view to show the Monolingual Corpora input template
+    Class-based view to show the input (monolingual) template
     """
-    template_name = 'researchdata/monolingual-corpora-input.html'
+    template_name = 'researchdata/input-monolingual.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['corpora_list'] = [
```

Comment:
This could do with a comment, explaining why only these languages are included here. Also, personally, I'd extract it into a constant at the top of the module, e.g. `MONOLINGUAL_CORPORA`